src/Util.hs

```diff
@@ -98,6 +98,14 @@ copyFile source target = do
     putProgressInfo $ renderAction "Copy file" source target
     copyFileChanged source target
 
+-- Same as copyFile, but not tracking the source as a build dependency
+copyFileUntracked :: FilePath -> FilePath -> Action ()
+copyFileUntracked source target = do
+    let dir = takeDirectory target
+    unlessM (liftIO $ IO.doesDirectoryExist dir) $ createDirectory dir
```

Comment:
Aha, looks like it fits indeed.

Are you suggesting to replace the line with `createDirectoryIfMissing True dir`?